src/quotas.md

```diff
@@ -99,3 +99,22 @@ applicable to the quota.
 |:------|:--------------------------------------------------------------------|:------:|:-------------------|
 | spend | Value of wallet-based payments or asset transfers from the account. |   Yes  | `monthly` `yearly` |
 | topup | Value of topups from bank accounts linked to the account.           |   Yes  | `monthly` `yearly` |
+
+<span id="quota-error-response">
```

Comment:
Could we close the span tag?

Could we use the word `-anchor` as a postfix to make it clear that this is for linking to?

Reply:
Done :)